types/svg-baker-runtime/src/browser-sprite.d.ts

```diff
@@ -0,0 +1,46 @@
+import Sprite from './sprite';
+import BrowserSpriteSymbol from './browser-symbol';
+
+import SpriteConfig from './sprite.config';
+import BrowserSpriteConfig from './browser-sprite.config';
+
+export default class BrowserSprite extends Sprite {
+    constructor(cfg?: SpriteConfig & BrowserSpriteConfig);
+
+    config: SpriteConfig & BrowserSpriteConfig;
```

Comment:
this should be on base class, either `protected` or `public` (default).
Are you sure about intersection type here?
https://github.com/JetBrains/svg-mixer/blob/68f4f77f780d64883e26719a882c9893888c7099/packages/svg-baker-runtime/src/sprite.js#L10

Reply:
SuperClass `Sprite` already has the type `config` and it was typed as `SpriteConfig`, this subclass would handle more config options by combining superclass's config and itself config.
You can see this https://github.com/JetBrains/svg-mixer/blob/68f4f77f780d64883e26719a882c9893888c7099/packages/svg-baker-runtime/src/browser-sprite.js#L37, subclass's config contains own `autoConfigure` field and  would be consumed by `this._autoConfigure` https://github.com/JetBrains/svg-mixer/blob/68f4f77f780d64883e26719a882c9893888c7099/packages/svg-baker-runtime/src/browser-sprite.js#L93